src/config/rds_aurora.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "Aurora RDS",
+  "displayName": "Amazon Aurora RDS",
+  "rules": [
+    {
+      "aws": {
+        "logGroup": "/aws/rds/cluster/%{DATA:resource_id}/%{GREEDYDATA:log_type}"
+      },
+      "sources": [
+        {
+          "sourceType": "logs",
+          "source": "log_group",
+          "condition": "$prefix('/aws/rds/cluster/')"
+        }
+      ],
+      "attributes": [
+        {
+          "key": "aws.service",
+          "pattern": "'rds'"
+        },
+        {
+          "key": "aws.resource.id",
+          "pattern": "resource_id"
+        },
+        {
+          "key": "aws.cluster.instance",
+          "pattern": "if(log_type == 'postgresql', &replace_regex(log_stream, '\\.0$', ''), &log_stream, @ )",
+          "priority": 9
+        },
+        {
+          "key": "aws.arn",
+          "pattern": "format_arn('arn:{}:rds:{}:{}:db:{}', [partition, region, account_id, \"aws.cluster.instance\"])",
+          "priority": 10
+        },
+        {
+          "key": "dt.source_entity",
+          "pattern": "dt_meid_rds_v2(\"aws.arn\")"
+        },
+        {
+          "key": "severity",
+          "pattern": "if(log_type == 'slowquery' || contains(log_content, '[Warning]'), &'WARNING', &if(contains(log_content, '[Error]'), &'ERROR', &'INFO', @), @)"
```

Comment:
This is tricky ground, let's assume we only have INFO logs then for each log we do 2 times check-content-run-on-whole-string. I am not sure if this should go inside without some basic perf test @mswiatkowska knows how to do them ;)
But I am open to other opinions, if you feel safe with this (remember we need to be able to do 1-10MB in minute) then fine

Reply:
I think this is time to create some automated perf test. But this is also tricky since CPUs are not made equal! ;)

Reply:
This is exactly the same pattern I've used for other rds databases. 
I'm only wondering - we will put proper Warning/Error severity for mysql slowquery and warning logs (and other logs containing [Error]/[Warning]) but e.g. postgres can produce some error logs like: 
`2021-08-09 10:08:56 UTC:10.0.0.123(52834):postgres@tstdb:[20175]:ERROR: column "wrong" does not exist at character 8`
and then we will classify this log as INFO.
Is it ok with us? Or maybe we want to just remove this level check as we are not able to classify all logs properly?

Reply:
thats true, in postgres severity is in `:` instead of `[]` ex. `:LOG:`

Reply:
we must decide if we lis add for postgresql or remove for all

Reply:
I splitted as we decided